packages/account/src/BaseSmartAccount.ts

```diff
@@ -55,10 +55,10 @@ export abstract class BaseSmartAccount implements IBaseSmartAccount {
   }
 
   async init(): Promise<this> {
+    if (this.entryPointAddress === DEFAULT_ENTRYPOINT_ADDRESS) return this;
```

Comment:
There should also be a check for the chain id against a default list of chain IDs.

Reply:
you mean list of chainIds where we knoe default EP is deployed for sure. and checking it against chainId stored in this class?

Reply:
Yes

Reply:
Chain configuration can be added later